README.md

```diff
@@ -30,6 +30,20 @@ cp config/config.sample.json public/config.json
 # edit public/config.json
 ```
 
+#### Allow to join group calls without camera and audio device:
+
+You can allow to join a group call without video and audio by setup this feature in your config.json:
+
+```json
+{
+  ...
+
+  "features": {
+    "feature_group_calls_without_video_and_audio": true
+  }
+}
+```
+
```

Comment:
Could this be moved to the end of the `Host it yourself` section? Also it would be good to have this under a `Features` section or similar